test/parallel/test-http-client-abort-no-agent.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+const net = require('net');
+
+const server = http.createServer(() => {
+  throw new Error('test invalidation');
```

Comment:
Even if I don't like `http.createServer(common.fail);`, for the reason stated above, I changed it to that for consistency as there are other tests which use the same pattern.